app/controllers/settings.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const { Controller, get, inject, computed } = Ember;
+
+export default Controller.extend({
+  session: inject.service(),
+  isAuthenticated: computed.oneWay('session.isAuthenticated'),
+
+  actions: {
+    signOut() {
```

Comment:
Hmm... I actually wouldn't worry about this _too_ much. In reality, the action of logging out is still housed by the session service; there might be a few different places that's called, but I think that's OK.